### PR TITLE
Walkthrough feedback misc fixes and changes

### PR DESCRIPTION
**Pull request by jsirianni**
### Proposed Change

Last week, the team walked through recent resource additions. The following changes are results from that meeting's findings.
- Set apache http default port to 80
- Update vcenter description
- Fix apache common icon
- Add descriptions to all destinations
- Jaeger: Hide ca cert param when tls verification is not enabled
- Fix haproxy caps
- Aerospike: ensure start_at depends on enable_logs
- Fix esxi spelling
- Fix wildfly default paths
- Kafka: Use new `createable` option to allow custom protocol versions
- Custom processor: Add new toggles for specifying telemetry types
- Add descriptions to all processors
- Use new `timezone` type instead of enum for all source types

##### Checklist
- [ ] Changes are tested
- [ ] CI has passed

### CODE REVIEW

**Comment by dsvanlani** [comment]
Verified:

~~Set apache http default port to 80~~
~~Update vcenter description~~
~~Fix apache common icon~~
~~Add descriptions to all destinations~~
Note: `Logging` destination does not have a description but its optional so I think thats okay.
~~Jaeger: Hide ca cert param when tls verification is not enabled~~
~~Fix haproxy caps~~
~~Aerospike: ensure start_at depends on enable_logs~~
~~Fix esxi spelling~~
~~Fix wildfly default paths~~
~~Kafka: Use new createable option to allow custom protocol versions~~
~~Custom processor: Add new toggles for specifying telemetry types~~
~~Add descriptions to all processors~~
~~Use new timezone type instead of enum for all source types~~

File: resources/processor-types/sample_log.yaml [hunk 1]
```diff
@@ -2,7 +2,8 @@ apiVersion: bindplane.observiq.com/v1beta
 kind: ProcessorType
 metadata:
   name: sampling
-  displayName: Sampling
+  displayName: Log Sampling
+  description: Sample logs with a configuration ratio.
```
Comment (dsvanlani) [inline resources/processor-types/sample_log.yaml:6]: I'm not sure about this description.  Is "configuration ratio" the correct term here?

File: resources/processor-types/delete_resource.yaml [hunk 2]
```diff
@@ -2,7 +2,8 @@ apiVersion: bindplane.observiq.com/v1beta
 kind: ProcessorType
 metadata:
   name: delete_resource
-  displayName: Delete Resource
+  displayName: Delete Resource Attribute
+  description: Delete resource attributes from metrics, traces, and logs.
```
Comment (dsvanlani) [inline resources/processor-types/delete_resource.yaml:6]: I don't think this description is necessary since it doesn't add any additional info.

**Review by dsvanlani — APPROVED** [review]
A couple nits that would be good for @ryancgoins to chime in on.  I don't think they should hold up merging though so I'm approving.

File: resources/processor-types/sample_log.yaml (hunk 1 above)
Comment (jsirianni) [reply resources/processor-types/sample_log.yaml]: No, it should be `configured` ratio. I pushed a change. I think we could do better, but I am not sure how to word it.

File: resources/processor-types/delete_resource.yaml (hunk 2 above)
Comment (jsirianni) [reply resources/processor-types/delete_resource.yaml:6]: I don't disagree but I think we want a description on all resources.

File: resources/destination-types/logzio.yaml [hunk 3]
```diff
@@ -4,6 +4,7 @@ metadata:
   name: logzio
   displayName: Logz.io
   icon: /icons/destinations/logzio.svg
+  description: Send metrics, logs and traces to Logz.io.
```
Comment (ryancgoins) [inline resources/destination-types/logzio.yaml:7]:
```suggestion
  description: Send metrics, logs, and traces to Logz.io.
```

**Review by ryancgoins — APPROVED** [review]

**Pull request merged**